web/html/src/components/config-channels.tsx

```diff
@@ -214,15 +146,11 @@ class ConfigChannels extends React.Component<ConfigChannelsProps, ConfigChannels
 
   search() {
     if (this.state.filter === this.state.search.filter) {
-      this.setState({
-        view: "search",
-      });
       return Promise.resolve();
```

Comment:
Since this is the only part of this branch now, I would recommend making the method async and removing this branch side altogether.

Reply:
Done